### ISSUE TITLE
A Mastering step with zip or double-metaphone matching fails with "no namespace" error

### ISSUE THREAD

**Issue by freeweel** [marklogic/marklogic-data-hub#2514]
Unable to run a flow with a mastering step containing zip or double-metaphone

What behavior are you expecting? My flow to complete without errors

Which Operating System are you using?  Windows 10 Pro

Which version of MarkLogic are you using? 9.0-7

Which version of the Data Hub Framework are you using? 5.0.0

Steps to re-create (done with QuickStart):

1. Create flow
1. Add a step of type "mastering"
1. Select a match using "double-metaphone" or "zip"
1. Run the flow

**NOTE:** 
When I manually added the namespace to my flow.json as shown below, the problem went away.  (The namespace was removed, however, if I modified the step in Quickstart).

```
 "algorithms" : {
      "algorithm" : [ {
      "name" : "double-metaphone",
      "namespace": "http://marklogic.com/smart-mastering/algorithms", 
      "function" : "double-metaphone",
      "at" : "/com.marklogic.smart-mastering/algorithms/double-metaphone.xqy"
      }, 
     ...
```

This is the error:

{"stack":"XDMP-IMPNULMODNS: (err:XQST0088) Cannot import module with null namespace\nin /com.marklogic.smart-mastering/algorithms/double-metaphone.xqy,\nin xdmp:function(fn:QName(\"\",\"double-metaphone\"), \"/com.marklogic.smart-mastering/algorithms/double-metaphone.xqy\")

**Comment by ayuwono** [comment]
Thanks @freeweel , tracking this issue on JIRA: https://project.marklogic.com/jira/browse/DHFPROD-2539

**Comment by aebadirad** [comment]
Fixed in 5.0.1
